NewSpring/ApollosProject
# [DEV-3969] Fix Content Item API

## DESCRIPTION

### What does this PR do, or why is it needed?
The need for this PR came about because I noticed an error when I tried to go to devotionals. They could not pull scripture correctly.

Turns out what was happening is that we had 2 folders in our API that were attempting to override the `content-item` API from Core. The first was the original one that I expected to be run. The second was a set of code that overrode the `content-item` API for the `WeekendContentItem`. The addition of the second set of code was writing over the first set of code. The result being that we ended up falling back to core functionality for all of the content items except for the extended part of the `WeekendContentItem`. 

This PR fixes our overridden `content-item` API. It merges the `WeekendContentItem` into the rest of the `content-item` API, giving us one overridden API instead of two. 

### How do I test this PR?
Make sure devotionals pull scripture correctly and that all of the rest of the content items work as they should.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
